proto/Makefile

```diff
@@ -23,6 +23,10 @@ proto: $(PROTO_GEN_SRC)
 	PROTOPATH=$(CURDIR) $(MAKE) -C service/fccontrol proto
 	PROTOPATH=$(CURDIR) $(MAKE) -C service/drivemount proto
 
+proto-docker:
+	docker run --rm \
+		-v $(CURDIR):/protobuf \
+		localhost/proto-builder
```

Comment:
This probably needs `localhost/proto-builder:$(DOCKER_IMAGE_TAG)` for the override specified in the other Makefile to work.

You may also need to add the `DOCKER_IMAGE_TAG?=latest` line.

Reply:
Good call, thanks!